### PR TITLE
cpu/kw2x: adapt for kinetis lpm

### PR DESCRIPTION
**Pull request by jfischer-no**
for kw2x

### CODE REVIEW

File: cpu/kw2x/include/cpu_conf.h [hunk 1]
```diff
@@ -72,6 +72,75 @@ extern "C"
...
+ * @brief LLWU module is always on.
+ */
+#define LLWU_UNLOCK()
+
+/**
+ * @brief Internal modules whose interrupts are mapped to LLWU wake up sources.
+ *
+ * Other modules CAN NOT be used to wake the CPU from LLS or VLLSx power modes.
+ */
+typedef enum llwu_wakeup_module {
```
Comment (jfischer-no) [inline cpu/kw2x/include/cpu_conf.h:106]: I think it is at all Kinetis M4 equal, some are reserved but the order is the same. Can we take it as a default?

File: cpu/kw2x/include/cpu_conf.h [hunk 2]
```diff
@@ -72,6 +72,75 @@ extern "C"
...
+    KINETIS_LPM_WAKEUP_MODULE_RTC_SECONDS = 7,
+    KINETIS_LPM_WAKEUP_MODULE_END,
+} llwu_wakeup_module_t;
+
+/**
+ * @brief enum that maps physical pins to wakeup pin numbers in LLWU module
+ *
+ * Other pins CAN NOT be used to wake the CPU from LLS or VLLSx power modes.
+ */
+typedef enum llwu_wakeup_pin {
```
Comment (jfischer-no) [inline cpu/kw2x/include/cpu_conf.h:123]: The same here.

**Comment by jnohlgard** [comment]
Should we move the LLWU stuff to `kinetis_common/llwu.[ch]`?

File: cpu/kw2x/include/cpu_conf.h [hunk 3]
```diff
@@ -72,6 +72,75 @@ extern "C"
...
+/**
+ * @name STOP mode bitfield values
+ * @{
+ */
+/** @brief Normal STOP */
+#define KINETIS_POWER_MODE_NORMAL       (0b000)
+/** @brief VLPS STOP */
+#define KINETIS_POWER_MODE_VLPS         (0b010)
+/** @brief LLS STOP */
+#define KINETIS_POWER_MODE_LLS          (0b011)
```
Comment (jnohlgard) [inline cpu/kw2x/include/cpu_conf.h:88]: are these bit combinations the same for all Kinetis CPUs?

**Pull request merged**